frontends/ol-forms/src/components/SelectField.tsx

```diff
@@ -14,7 +14,7 @@ export interface Option {
 
 interface Props {
   name: string
-  value: null | undefined | string | string[]
+  value?: null | string | string[]
```

Comment:
`value: undefined ... ` means that `value` can be explicitly assigned `undefined`: `{ value: undefined }`, but does NOT accept missing properties (like `{}`).

So when a property is optional, we generally want `propName?: PropType`. 

Javascript... :sigh: